oe_multilingual.info.yml

```diff
@@ -0,0 +1,42 @@
+name: Drupal module template
```

Comment:
Should we update this to something more appropriate ?
Same goes for the description hereunder ?